### PR TITLE
Add disable logging option to fix oom

### PR DESCRIPTION
**Pull request by li-wu**
Add `--disable-logging` option to disable the logging to address the oom issue.

I also removed `structlog` which does not have interface to change handler.

### CODE REVIEW

**Comment by li-wu** [comment]
The memory usage is less than 100MB per process with logging disable:
```
bash-5.0# ps -o pid,user,vsz,rss,comm,args
PID   USER     VSZ  RSS  COMMAND          COMMAND
    1 root     2212 1712 entrypoint.sh    {entrypoint.sh} /bin/bash /sbin/entrypoint.sh standalone
    8 root     4264  496 sshd             /usr/sbin/sshd
    9 root     1.1g  67m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
   10 root     1552    4 tail             tail -F -n0 /etc/hosts
 1350 root     1.0g  59m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1460 root     1.1g  89m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1461 root     1.1g  89m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1462 root     1.1g  90m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1472 root     1.1g  89m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1473 root     1.1g  87m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1474 root     1.1g  89m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1478 root     1.1g  90m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1487 root     1.1g  88m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1489 root     1.1g  89m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1490 root     1.1g  87m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1491 root     1.1g  90m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1504 root     1.1g  86m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1505 root     1.1g  86m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1506 root     1.1g  87m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1513 root     1.1g  86m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1517 root     1.1g  90m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1518 root     1.1g  87m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1528 root     1.1g  89m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1529 root     1.1g  88m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
 1530 root     1.1g  86m splunk_eventgen  {splunk_eventgen} /usr/bin/python2 /usr/bin/splunk_eventgen service --role standalone
13941 root     2404 2204 bash             bash
15406 root     1556    4 ps               ps -o pid,user,vsz,rss,comm,args
```

File: splunk_eventgen/__main__.py [hunk 1]
```diff
@@ -53,6 +53,8 @@ def parse_args():
     generate_subparser.add_argument(
         "--generator-queue-size", type=int, default=500, help="the max queue size for the "
         "generator queue, timer object puts all the generator tasks into this queue, default max size is 500")
+    generate_subparser.add_argument("--disable-logging", action="store_true",
+                                    help="disable logging")
```
Comment (GordonWang) [inline splunk_eventgen/__main__.py]: Probably put the default value as False explicitly as some kind of document.

File: splunk_eventgen/eventgen_core.py [hunk 2]
```diff
@@ -240,17 +240,22 @@ def _create_generator_workers(self, workercount=20):
             self.workerPool = []
             for worker in range(workercount):
                 # builds a list of tuples to use the map function
+                disable_logging = True if self.args and self.args.disable_logging else False
```
Comment (GordonWang) [inline splunk_eventgen/eventgen_core.py]: if we set the default value in arg parser, no need to use if else here. Just use
`disable_logging = self.args.disable_logging`

Comment (li-wu) [reply splunk_eventgen/eventgen_core.py]: `self.args` might be `None` here.

File: splunk_eventgen/__main__.py (hunk 1 above)
Comment (li-wu) [reply splunk_eventgen/__main__.py]: agree. I tried to find a suitable place for this info but it seems all other sub-command does not have this kind of doc either.

**Pull request merged**